test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4187,6 +4190,27 @@ def tracejob_match(self, msg, id=None, n=50, tail=True,
                                regexp, max_attempts, interval, starttime,
                                endtime, level, existence)
 
+    def del_hooks_nodes_qs(self, del_hooks=True,
+                           del_nodes=True,
+                           del_qs=True,
+                           del_vdef=True):
+        """
+        Delete nodes, queues and hooks
+        """
```

Comment:
please document the function's arguments in the doctring

Reply:
Added function arguments as docstring